src/jimgw/single_event/transforms.py

```diff
@@ -20,45 +24,70 @@
 
 
 @jaxtyped(typechecker=typechecker)
-class ComponentMassesToChirpMassMassRatioTransform(BijectiveTransform):
+class SpinToCartesianSpinTransform(NtoNTransform):
```

Comment:
I see a lot of new transforms in the code below. Are they relevant to this PR? If not, can we separate them into a different PR for traceability?

Reply:
These are no new transforms.I just might have shuffled around their positions a bit (by accident) which is why they show up in the diffs.

Reply:
Okay. We can proceed then